Content.Shared/Preferences/HumanoidCharacterProfile.cs

```diff
@@ -232,7 +232,7 @@ public HumanoidCharacterProfile WithAntagPreference(string antagId, bool pref)
         }
 
         public string Summary =>
-            $"{Name}, {Age} years old {Sex.ToString().ToLower()} human.";
+            $"{Name}, {Age} years old human. Their gender is {Sex.ToString().ToLower()}.";
```

Comment:
Not your code but how does the sex part go with localization?

Reply:
Idk